crates/cargo-gpu/src/spirv_source.rs

```diff
@@ -204,124 +204,57 @@ impl SpirvSource {
         let canonical_shader_path = shader_crate_path.to_path_buf();
         Self::shader_crate_path_canonical(&mut canonical_shader_path.clone())?;
 
-        log::debug!(
-            "Running `cargo tree` on {}",
-            canonical_shader_path.display()
-        );
-        let output_cargo_tree = std::process::Command::new("cargo")
-            .current_dir(canonical_shader_path.clone())
-            .args(["tree", "--workspace", "--prefix", "none"])
-            .output()?;
-        anyhow::ensure!(
-            output_cargo_tree.status.success(),
-            format!(
-                "could not query shader's `Cargo.toml` for `spirv-std` dependency: {}",
-                String::from_utf8(output_cargo_tree.stderr)?
-            )
-        );
-        let cargo_tree_string = String::from_utf8_lossy(&output_cargo_tree.stdout);
+        log::debug!("Running `cargo metadata` on {}", canonical_shader_path.display());
```

Comment:
It might be more accurate to say that we're running `cargo_metadata::MetadataCommand` now right?

Reply:
It was to follow this line
https://github.com/Rust-GPU/cargo-gpu/pull/66/files#diff-e50e99da00483f62292c3fcb50fdea5e75b82a3d8506d159cc26e1bca432c282L208
It says it's running `cargo tree` like terminal command and I believe it should still print it this way?